ios/Exponent/Kernel/Views/Loading/EXManagedAppSplashScreenConfigurationBuilder.m

```diff
@@ -0,0 +1,98 @@
+#import <UIKit/UIKit.h>
+
+#import "EXManagedAppSplashScreenConfigurationBuilder.h"
+#import "EXUtil.h"
+
+
+static const NSString *kManifestIosKey = @"ios";
+static const NSString *kManifestSplashKey = @"splash";
+static const NSString *kManifestResizeModeKey = @"resizeMode";
+static const NSString *kManifestBackgroundColorKey = @"backgroundColor";
+static const NSString *kManifestTabletImageUrlKey = @"tabletImageUrl";
+static const NSString *kManifestImageUrlKey = @"imageUrl";
+
+static const NSString *kImageResizeModeContain = @"contain";
+static const NSString *kImageResizeModeCover = @"cover";
+
+@implementation EXManagedAppSplashScreenConfigurationBuilder
+
++ (EXSplashScreenConfiguration *)parseManifest:(NSDictionary *)manifest
+{
+  UIColor *backgroundColor = [[self class] parseBackgroundColor:manifest];
+  NSString *imageUrl = [[self class] parseImageUrl:manifest];
+  EXSplashScreenImageResizeMode imageResizeMode = [[self class] parseImageResizeMode:manifest];
+  return [[EXSplashScreenConfiguration alloc] initWithBackgroundColor:backgroundColor
+                                                             imageUrl:imageUrl
+                                                      imageResizeMode:imageResizeMode];
+}
+
++ (UIColor * _Nonnull)parseBackgroundColor:(NSDictionary *)manifest
+{
+  // TODO: (@bbarthec) is backgroundColor always in HEX format on iOS? (most possibly it is not)
```

Comment:
That's what we advise in the docs

https://docs.expo.io/versions/latest/config/app/#backgroundcolor-4

Reply:
Yeah, for now it's true, but it's unnecessary limitation for the `splash-screen` module. I'll adjust the comment.